ArkSavegameToolkitNet/ArkSaveGame.cs

```diff
@@ -126,6 +126,95 @@ public bool LoadEverything()
 
             readBinary(_archive, _mmf);
 
+            //Now parse out cryo creature data
+            var cryoPodEntries = Objects.Where(o => o.ClassName.Name == "PrimalItem_WeaponEmptyCryopod_C" && o.GetProperty<PropertyArray>(ArkName.Create("CustomItemDatas")) != null).ToList();
+
+            IList<GameObject> cryoCreatureObjects = new List<GameObject>();
+
+            foreach(var cryoEntry in cryoPodEntries)
+            {
+                try
+                {
+                    var customData = cryoEntry.GetProperty<PropertyArray>(ArkName.Create("CustomItemDatas"));
+                    if (customData != null && customData.Value.Count > 0)
+                    {
+                        StructPropertyList customProperties = (StructPropertyList)customData.Value[0];
+                        var byteProperty = customProperties.GetProperty<PropertyStruct>("CustomDataBytes");
+                        if (byteProperty != null)
+                        {
+                            try
+                            {
+                                StructPropertyList byteArrayProperty = (StructPropertyList)byteProperty.Value;
+                                StructPropertyList byteArrayList = (StructPropertyList)byteArrayProperty.GetProperty<PropertyArray>("ByteArrays").Value[0];
+                                var byteList = byteArrayList.GetProperty<PropertyArray>("Bytes");
+                                var byteListValue = byteList.Value;
+
+                                //convert from sbytes to bytes 
+                                byte[] cryoDataBytes = new byte[byteListValue.Count];
+                                for (int i = 0; i < cryoDataBytes.Length; i++)
+                                {
+                                    int.TryParse(byteListValue[i].ToString(), out int byteValue);
```

Comment:
What are you trying to do here exactly?  

you can directly cast an `sbyte` to `int` (which is a signed integer), `(int)byteListValue[i]`... but I don't see how that is relevant to converting from `sbyte` to `byte`, or necessary.  

casting this `int` to `byte` (`(byte)byteValue`) will cause an exception if it is negative, the `TryParse` of a negative signed byte to a signed integer isn't going to handle that concern.

Reply:
Thanks for the feedback brcaswell.  I'll be honest I'm more used to working with business applications at a high level with a database engine so I've just hacked this together to serve my own purposes. 

I will review your suggestions when I get home this evening and make the appropriate corrections.